resource-watch/resource-watch
# Back office: Explore form

![image](https://user-images.githubusercontent.com/545342/76766380-2f088980-6798-11ea-91f2-177a1389b092.png)

## Overview
This PR adds a new Explore form to the back office.

## Testing instructions
Go to http://localhost:9000/admin/data/explore and verify that the different sections work properly.

⚠️ **NOTE:** We're still waiting for the API to implement the `isHighlighted` property for datasets together with the `config` service. Thus, this form isn't yet `usable` from the data perspective
